### ISSUE TITLE
Delete this fork?

### ISSUE THREAD

**Issue by katyhuff** [arfc/recycle#32]
Should we delete this fork @gtw2 . I see you recently pulled something, but I thought this fork is basically pointless since you're making PRs directly to cyclus/recycle. Thoughts?

**Comment by gtw2** [comment]
I agree, but the current PR into cyclus/recycle is coming from this fork - I'm awaiting Baptiste again but when that gets merged this should be deleted

**Comment by katyhuff** [comment]
Ah.

**Comment by gtw2** [comment]
@katyhuff this can be deleted now I believe - I lack the power to do so, however.
